build.xml

```diff
@@ -0,0 +1,110 @@
+<project name="sonatype-restlet" default="sonatype-dist">
```

Comment:
because trying to follow the instructions in the readme manually copying only certain dependencies from the produced maven 2 repo is error prone, and I felt that with a build.xml, we both document the required steps and implement them to clearly see what is being done.

At the end of the day, this is just a small wrapper around invocating the real build.xml, and then producing the zip file that sonatype needs.